priv/templates/nimble_template/lib/otp_app_web/controllers/error_json.ex.eex

```diff
@@ -39,17 +71,13 @@ defmodule <%= web_module %>.Api.ErrorView do
     "#{sentence} and #{List.last(messages)}"
   end
 
-  defp build_error_response(code: code, detail: detail, message: message) do
-    %{
-      errors: [
-        %{
-          code: code,
-          detail: detail,
-          message: message
-        }
-      ]
-    }
-  end
-
   defp translate_errors(changeset), do: Changeset.traverse_errors(changeset, &translate_error/1)
+
+  defp translate_error({msg, opts}) do
+    if count = opts[:count] do
```

Comment:
```suggestion
    if count == opts[:count] do
```

Reply:
It assigns count at the same time so it's correct.

See original implementation here:
https://github.com/nimblehq/elixir-templates/blob/25fd192d8810348e9a0ba2579b4616f2d2952107/priv/templates/nimble_template/lib/otp_app_web/views/error_helpers.ex.eex#L43-L48

This originates from Phoenix as well

Reply:
shall we mark this one as `Resolved` @nvminhtue